velox/docs/functions/spark/datetime.rst

```diff
@@ -49,6 +49,14 @@ These functions support TIMESTAMP and DATE input types.
     ``day`` need to be from 1 to 31, and matches the number of days in each month.
     days of ``year-month-day - 1970-01-01`` need to be in the range of INTEGER type.
 
+.. spark:function:: next_day(start_date, dayofweek) -> date
```

Comment:
Please explain what `dayofweek` is valid input.

Reply:
update